src/test/csi-qingcloud/values.yaml

```diff
@@ -52,6 +52,13 @@ controller:
     requests:
       memory: 100Mi
       cpu: 100m
+  vpa:
+    minAllowed:
+      cpu: 100m
+      memory: 100Mi
+    maxAllowed:
+      cpu: 500m
+      memory: 500Mi
```

Comment:
all maxAllowed values look small except this one. Make them as 10 * requests.limits may be better.

Reply:
Okay